README.md

```diff
@@ -8,8 +8,36 @@
 
 Python bindings for [DART][dart], the Dynamic Animation and Robotics Toolkit.
 
+## Installation
 
-## Dependencies
+### On Ubuntu using `apt-get`
+
+#### Artful, Zesty
+> Installing `dartpy` on Xenial using `apt-get` is not available for now, but we're working on it.
```

Comment:
What's wrong with `Xenial` support at the moment?

Reply:
I didn't investigate this further, but only the GCC in Xenial [complains building `dartpy` with DART](https://launchpadlibrarian.net/347611882/buildlog_ubuntu-xenial-amd64.python-dartpy_0.0.1.ppa0~xenial_BUILDING.txt.gz).

Reply:
```
/<<PKGBUILDDIR>>/src/generated/_ZN4dart6common11make_uniqueINS0_13MakeCloneableINS0_6Aspect10PropertiesENS_8dynamics6detail15JointPropertiesEEEJRKS8_EEESt10unique_ptrIT_St14default_deleteISC_EEDpOT0_.cpp:25:1: error: ‘std::unique_ptr<T> dart::common::make_unique(Args&& ...) [with T = dart::common::MakeCloneable<dart::common::Aspect::Properties, dart::dynamics::detail::JointProperties>; Args = {const dart::common::MakeCloneable<dart::common::Aspect::Properties, dart::dynamics::detail::JointProperties>&}]’ conflicts with a previous declaration
 }
 ^
/<<PKGBUILDDIR>>/src/generated/_ZN4dart6common11make_uniqueINS0_13MakeCloneableINS0_6Aspect10PropertiesENS_8dynamics6detail15JointPropertiesEEEJRKS8_EEESt10unique_ptrIT_St14default_deleteISC_EEDpOT0_.cpp:18:6: note: previous declaration ‘void _ZN4dart6common11make_uniqueINS0_13MakeCloneableINS0_6Aspect10PropertiesENS_8dynamics6detail15JointPropertiesEEEJRKS8_EEESt10unique_ptrIT_St14default_deleteISC_EEDpOT0_()’
 void _ZN4dart6common11make_uniqueINS0_13MakeCloneableINS0_6Aspect10PropertiesENS_8dynamics6detail15JointPropertiesEEEJRKS8_EEESt10unique_ptrIT_St14default_deleteISC_EEDpOT0_()
      ^
In file included from /usr/include/dart/common/Memory.hpp:195:0,
                 from /usr/include/dart/math/MathTypes.hpp:41,
                 from /usr/include/dart/math/Geometry.hpp:38,
                 from /usr/include/dart/dynamics/Shape.hpp:41,
                 from /usr/include/dart/dynamics/ConeShape.hpp:35,
                 from /usr/include/dart/dynamics/dynamics.hpp:3,
                 from /<<PKGBUILDDIR>>/include/dartpy/pointers.h:4,
                 from /<<PKGBUILDDIR>>/src/generated/_ZN4dart6common11make_uniqueINS0_13MakeCloneableINS0_6Aspect10PropertiesENS_8dynamics6detail15JointPropertiesEEEJRKS8_EEESt10unique_ptrIT_St14default_deleteISC_EEDpOT0_.cpp:1:
/usr/include/dart/common/detail/Memory-impl.hpp:65:20: note: a later -fabi-version= (or =0) avoids this error with a change in mangling
 std::unique_ptr<T> make_unique(Args&&... args)
```

> **note: a later -fabi-version= (or =0) avoids this error with a change in mangling**
Maybe we just need to add that flag?

Reply:
[New build with that option](https://launchpad.net/~personalrobotics/+archive/ubuntu/ppa/+build/13806086) :crossed_fingers:

Reply:
Successfully built `dartpy` on Xenial using `-fabi-version=6`!